Makefile

```diff
@@ -43,7 +43,7 @@ all:
 	$(MAKE) test
 
 install:
-	$(PYTHON) setup.py install
+	$(PIP) install '.[crypto]'
```

Comment:
I wonder if this installs the package in the `pynuodb` namespace, or if it installs everything inside of this so that it can be imported directly.